help.js

```diff
@@ -107,7 +105,7 @@ module.exports = {
         },
         seq: {
           type: 'SequenceNumber',
-          optional: false,
+          optional: true,
```

Comment:
Fixing this unbreaks [`ssb createHistoryStream`](https://github.com/fraction/ssb-cli#usage). :tada: